tools/istio-iptables/pkg/dependencies/implementation.go

```diff
@@ -155,3 +155,9 @@ func (r *RealDependencies) RunQuietlyAndIgnore(cmd string, args ...string) {
 		_ = r.execute(cmd, true, args...)
 	}
 }
+
+// CmdOutput runs a command and returns its standard output
+func (r *RealDependencies) CmdOutput(cmd string, args ...string) (string, error) {
```

Comment:
Can you reuse the existing Run command? It has some extra logic dealing with XTables command output etc. I think you can argument the function to return stdout.

Reply:
Ah, I just saw the changes in your PR  #33825. I resued the existing Run method to get stdout. I hope it's better now, but considering howardjohn's comment and thinking how to proceed now..